src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,362 @@
+      />
+    )
+  }
+
+  private renderItems(variables, getItemProps, highlightedIndex) {
+    const { items } = this.props
+    if (items.length > 0) {
+      return items.map((item, index) => {
+        return (
+          <ListItem
+            key={item.key}
+            content={item.content}
+            header={item.header}
+            media={<Image src={item.image} avatar />}
+            variables={{
+              ...(highlightedIndex === index && {
+                headerColor: variables.listItemTextColor,
+                contentColor: variables.listItemTextColor,
+              }),
+            }}
+            styles={{
+              backgroundColor:
+                highlightedIndex === index
+                  ? variables.listItemHighlightedBackgroundColor
+                  : variables.listItemBackgroundColor,
+            }}
+            {...getItemProps({ index, item })}
+          />
+        )
+      })
+    }
+    // render no match error.
+    return [
+      {
+        key: 'people-picker-no-results-item',
+        content: <Text weight="bold" content={`We couldn't find any matches.`} />,
+      },
+    ]
+  }
+
+  private renderActive(styles) {
+    const active = this.state.active
+
+    return active.length === 0
+      ? null
+      : active.map((item, index) => (
+          <Label
+            role="presentation"
+            styles={styles.activeListLabel}
+            circular
+            key={`active-item-${index}`}
+            content={item.header}
+            image={{ src: item.image, avatar: true }}
+            icon={{
+              name: 'close',
+              onClick: this.onCloseIconClick.bind(this, item),
+              onKeyDown: this.onCloseIconKeyDown.bind(this, item),
+              'aria-label': `Remove ${item.header} from selection.`,
+              'aria-hidden': false,
+              role: 'button',
+            }}
+          />
+        ))
+  }
+
+  onInputFocus = () => {
+    this.setState({ focused: true })
+  }
+
+  onInputBlur = () => {
+    this.setState({ focused: false })
+  }
+
+  onContainerClick = isOpen => {
+    !isOpen && this.inputRef.focus()
+  }
+
+  onInputKeyUpIfMultiple = event => {
+    const { inputValue, active, backspaceDelete } = this.state
+
+    switch (keyboardKey.getCode(event)) {
+      case keyboardKey.Backspace:
+        if (inputValue === '' && active.length > 0) {
+          if (!backspaceDelete) {
+            this.setState({ backspaceDelete: true })
+          } else {
+            this.removeFromActive()
+          }
+        }
+      default:
+        return
+    }
+  }
+
+  onDropdownChange = element => {
+    if (this.props.multiple) {
+      const newActive = [...this.state.active, element]
+
+      _.invoke(this.props, 'onChange', newActive)
+
+      this.setState({
+        active: newActive,
+        inputValue: '',
+        message: `${element.header} has been selected.`,
```

Comment:
as we will need to be able to translate this text, we should probably pass a callback and evaluate it here, something like
```
message: props.translateSelectedMessage ? props.translateSelectedMessage(element.header) : `${element.header} has been selected.`
```

Reply:
added the following props:
`  getA11yStatusMessage?: (options: A11yStatusMessageOptions<DropdownListItem>) => string
  getA11ySelectedMessage?: (item: DropdownListItem) => string
  getA11yRemovedMessage?: (item: DropdownListItem) => string
  getA11yRemoveItemMessage?: (item: DropdownListItem) => string
  getNoResultsMessage?: () => string
`